src/views/MyProfile/Components/VictoryPromiseDisplay/index.js

```diff
@@ -223,118 +228,114 @@ export default class VictoryPromiseDisplay extends React.Component {
-                          </React.Fragment>
-                        }
-                      >
-                        {IMG_CC}
-                      </HoverText>
-                      <HoverText
-                        title={
-                          <React.Fragment>
-                            <Typography color="inherit">Intellectual Maturity</Typography>
-                            {"It's very engaging. Right?"}
-                          </React.Fragment>
-                        }
-                      >
-                        {IMG_IM}
-                      </HoverText>
-                    </Grid>
-                    <Grid container justify="center">
-                      <HoverText
-                        title={
-                          <React.Fragment>
-                            <Typography color="inherit">Lives of Service</Typography>
-                            {"It's very engaging. Right?"}
-                          </React.Fragment>
-                        }
-                      >
-                        {IMG_LS}
-                      </HoverText>
-                      <HoverText
-                        title={
-                          <React.Fragment>
-                            <Typography color="inherit">Leadership Worldwide</Typography>
-                            {"It's very engaging. Right?"}
-                          </React.Fragment>
-                        }
-                      >
-                        {IMG_LW}
-                      </HoverText>
-                    </Grid>
-                  </Grid>
-                </CardContent>
-              </Card>
-            </Grid>
+        <Grid item xs={12} container justify="center">
+          <Grid>
+            <HoverText
+              title={
+                <React.Fragment>
+                  <Typography color="inherit">Christian Character</Typography>
+                  {
+                    'Opportunities encouraging faith formation and its connection to living, learning and leading with others'
+                  }
+                </React.Fragment>
+              }
+            >
+              {IMG_CC}
+            </HoverText>
+            <HoverText
+              title={
+                <React.Fragment>
+                  <Typography color="inherit">Intellectual Maturity</Typography>
+                  {
+                    'Opportunities to extend critical reasoning, deepen understanding, and ignite imagination'
+                  }
+                </React.Fragment>
+              }
+            >
+              {IMG_IM}
+            </HoverText>
           </Grid>
-        </section>
+          <Grid container justify="center">
+            <HoverText
+              title={
+                <React.Fragment>
+                  <Typography color="inherit">Lives of Service</Typography>
+                  {
+                    "Opportunities to lend one's strengths and talents with our partners to our neighbors"
```

Comment:
Is this text exactly how Chris Carlson wanted it, or was it your own description?